### PR TITLE
bar: modernise and add test

### PR DESCRIPTION
**Pull request by DomT4**
I’m pretty sure the bot is going to yell at me for not spacing out the test system call, but AFAIK I’m stuck with doing so given the file into file ` > ` action.

### CODE REVIEW

File: Library/Formula/bar.rb [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
   def install
-    bin.install 'bar'
+    bin.install "bar"
+  end
+
+  test do
+    (testpath/"test1").write "pumpkin"
+    (testpath/"test2").write "latte"
+    system bin/"bar test1 test2 > test3"
+    assert File.read("test3").include?("latte")
```
Comment (mikemcquaid) [inline Library/Formula/bar.rb]: Could just use `system_output` instead of writing to `test3` here.

Comment (DomT4) [inline Library/Formula/bar.rb]: Not super-sure how to use ` system_output ` in this context. Happy to change if you don't mind nudging me in the right direction.

Comment (xu-cheng) [inline Library/Formula/bar.rb]: It should be `assert shell_output("#{bin}/bar test1 test2").include? "latte"`

Comment (DomT4) [inline Library/Formula/bar.rb]: Thanks, I thought that too, but the language threw me off a bit. I'll check it out shortly.

Comment (DomT4) [inline Library/Formula/bar.rb]: Yup, works. Cheers for the nudge, have pushed :+1:

File: Library/Formula/bar.rb [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 
   def install
-    bin.install 'bar'
+    bin.install "bar"
+  end
+
+  test do
+    (testpath/"test1").write "pumpkin"
+    (testpath/"test2").write "latte"
+    assert shell_output("#{bin}/bar test1 test2").include? "latte"
```
Comment (mikemcquaid) [inline Library/Formula/bar.rb]: `assert_match /latte/, shell_output("#{bin}/bar test1 test2")`

Comment (DomT4) [inline Library/Formula/bar.rb]: Thanks. Pushed! My text editor always throws tantrums about ` /latte/ ` in Ruby code. Not sure why.

**Pull request closed without merging**